README.md

```diff
@@ -1,79 +1,81 @@
-provided by Colony. It comes with a sample 'demo' pipeline, so that you will have an entrypoint from where
-you can start modeling your CI/CD flows using Jenkins and Colony  
+After deployment, this Jenkins instance can be immediately used to launch your pipelines against environments
+provided by Colony. It comes with a sample pipeline, so that you will have an entry point from which
+you can start modeling your CI/CD flows using Jenkins and Colony.
 
-### Repo consists of two blueprints:
+### This repo is comprised of two blueprints:
 
-**Jenkins**
+#### Jenkins
 
-This blueprint deploys a Sandbox with Jenkins itself. Flow is the following:
+This blueprint deploys a Sandbox with Jenkins itself. Its flow is comprises the following steps:
 
-* It first deploys jenkins
-* Installs colony jenkins plugin and it's dependencies
-* Configures connection between Colony and Jenkins
-* Creates a sample demo pipeline 
+1. Deploy Jenkins
+2. Install Colony's Jenkins plugin and its dependencies
+3. Configure connection between Colony and Jenkins
+4. Create a sample demo pipeline 
 
-**Promotions Manager**
+#### Promotions Manager
 
-It's a demo blueprint which is used by the pre-defined pipeline in Jenkins.
-It deploys a sandbox with sample app consisting of backend, frontend and database.
+A demo blueprint used by the predefined pipeline in Jenkins.
+Deploys a sandbox with a sample app consisting of back-end, front-end, and database.
 
-## How To Use
+## Usage
 
-### prerequisites:
+### prerequisites
 
 * Only AWS cloud provider is currently supported
-* Before starting using this solution you need to have an S3 bucket created to store artifacts built by the sample Jenkins job
+* Before using this solution, you need to have an S3 bucket created to store artifacts built by the sample Jenkins job
 * You also need to create an IAM role (for EC2 service) and configure it with write permissions to your S3 bucket
 
 ### Colony setup
 
-#### Link you AWS account and S3 bucket to Colony
+#### Link your AWS account and S3 bucket to Colony
 
-Go to **Settings** and link your aws account on **Cloud Account** page and set your s3 bucket as an artifacts repository
-on **Integrations** page
+Go to **Settings** and link your aws account on **Cloud Account** page. Then, set your s3 bucket as an artifacts repository
+on **Integrations** page.
 
 #### Connect blueprint repo to your Colony space
 
-In your space go to **Settings -> Blueprint Repositories** and connect this repo as a blueprint repository for Sandbox Environment.
-If you want to do some experiments with code and make some changes fork it and connect your own repository
+In your space, go to **Settings -> Blueprint Repositories** and connect this repo as a blueprint repository for Sandbox Environment.
+If you want to experiment with the code and make some changes, fork it and connect your own repository.
```

Comment:
...for the repository and connect your own one.

Reply:
I don't understand what needs to be changed. While were at it, feel free to make the changes yourself and/or totally ignore mine - I'm not a native English speaker and just wanted to point to points/areas that need improvement IMHO.

Reply:
my mistake. disregard my comment